src/sketchup-stl.rb

```diff
@@ -52,7 +52,7 @@ def self.translate(string)
       'This is an open source project sponsored by the SketchUp team. More ' <<
       'info and updates at https://github.com/SketchUp/sketchup-stl'
     )
-    extension.version = '2.1.6'
```

Comment:
Well, I've tagged the commit in my repo but not sure what to do now.  Just need to tag commit #fefa9098be9600efb5ac7e49ef52777cc973fe76 with v2.1.7